lib/tests.nix

```diff
@@ -58,6 +58,7 @@ let
               "mkTestDerivationFromNixvimModule: the `dontRun` argument is deprecated. You should use the `test.runNvim` module option instead."
               { config.test.runNvim = !dontRun; }
           ))
+          { wrapRc = true; }
```

Comment:
Should this also be included in the `.extend {}` in `mkTestDerivationFromNvim` above?

We don't use that function internally, and anyone downstream _should_ already have a wrapped RC because their `nvim` will have been built by the standalone wrapper, so it's probably not strictly necessary there...

Reply:
I don't really see how you would be able to call that function without having a standalone module

Reply:
They could use `evalNixvim`, if if they're really fancy they could manually call `evalModules`.

More reasonably, a home-manager user could decide to use the `config.programs.nixvim.test.derivation` from their home-manager configuration.

Reply:
Ah my mistake, I was talking about how they could get the test _without_ using either of these functions...

You're right, regarding my original comment. It's very unlikely they'd get a standalone nixvim derivation without using the standalone wrapper!